src/core/src/bootstrap/Bootstrapper.py

```diff
@@ -122,6 +122,8 @@ def basic_environment_health_check(self):
         self.composite_logger.log("Python version: " + " ".join(sys.version.splitlines()))
         self.composite_logger.log("Linux distribution: " + str(self.env_layer.platform.linux_distribution()) + "\n")
 
+        self.env_layer.ensure_tty_not_required()
+
         # Ensure sudo works in the environment
         sudo_check_result = self.env_layer.check_sudo_status()
```

Comment:
It may make sense for both of these to actually be in the handler so the entire execution flow benefits from it long term.

Reply:
(Sudo check may be left behind, and possibly duplicated).

Reply:
Resolved. Added EnvLayer.py and EnvManager.py in handler. EnvManager has check_sudo_status, tty mitigation and checking for mitigation. EnvLayer contains check for current tty settings on the machine. Also, copied a few other functions, that are required, from EnvLayer in Core to Handler